source/wp-content/themes/wporg-developer-2023/scss/prism.scss

```
@@ -210,7 +210,7 @@ pre.line-numbers > code {
 /* Copy button */
 .wp-code-block-button-container {
 	position: sticky;
-	top: calc(var(--wp-global-header-height) + var(--wp-admin--admin-bar--height));
+	top: calc(var(--wp--custom--local-navigation-bar--spacing--height) + var(--wp-admin--admin-bar--height));
```

I noticed this doesn't work when I'm logged out because `--wp-admin--admin-bar--height` is undefined, might need a fallback
```suggestion
	top: calc(var(--wp--custom--local-navigation-bar--spacing--height) + var(--wp-admin--admin-bar--height, 0));
```

---

Good catch! Fixed.

---

Good catch! Fixed.